apps/dsaparam.c

```diff
@@ -123,9 +123,6 @@ int dsaparam_main(int argc, char **argv)
     in = bio_open_default(infile, 'r', informat);
     if (in == NULL)
         goto end;
-    out = bio_open_owner(outfile, outformat, private);
```

Comment:
it is used before statement if (!noout ) ..

Reply:
but one thing I am confused about this is: do the options as '-list_curves', '-text' and '-modulus', etc need to write the content to the file specified by '-out' option? It seems the '-out' options are mainly used for outputting DER/PEM formatted keys/certificates/CSRs... in many apps.

Reply:
Hmm, yes, but this should probably stay, because of binary compatibility.
But what to do about ASN1 output format, that will output text followed by DER format.

Reply:
So this part could be considered in next release after the 1.1.1 stable branch is create, right?